.ci/pytorch/test.sh

```diff
@@ -136,6 +136,13 @@ if [[ "$BUILD_ENVIRONMENT" == *rocm* ]]; then
   rocminfo | grep -E 'Name:.*\sgfx|Marketing'
 fi
 
+if [[ "$BUILD_ENVIRONMENT" == *xpu* ]]; then
+  # shellcheck disable=SC1091
+  source /opt/intel/oneapi/compiler/latest/env/vars.sh
```

Comment:
Looks like its the script described by this doc:
https://www.intel.com/content/www/us/en/docs/oneapi/programming-guide/2023-0/use-the-setvars-script-with-linux-or-macos.html

Could you please add some comments to what it does and link to documentation

Reply:
Added comments and reference url